plugins/modules/nodebalancer_stats.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+"""This module allows users to view Nodebalancer Stats."""
+
+from __future__ import absolute_import, division, print_function
+
+from typing import Any, List, Optional
+
+from ansible_collections.linode.cloud.plugins.module_utils.doc_fragments import (
+    nodebalancer_stats as docs,
+)
+from ansible_collections.linode.cloud.plugins.module_utils.linode_common import (
+    LinodeModuleBase,
+)
+from ansible_collections.linode.cloud.plugins.module_utils.linode_docs import (
+    global_authors,
+    global_requirements,
+)
+from ansible_specdoc.objects import (
+    FieldType,
+    SpecDocMeta,
+    SpecField,
+    SpecReturnValue,
+)
+
+traffic_spec = {
+    "in": SpecField(
+        type=FieldType.list,
+        element_type=FieldType.integer,
+        description=[
+            "An array of key/value pairs representing unix timestamp"
+            + " and reading for inbound traffic."
+        ],
+    ),
+    "out": SpecField(
+        type=FieldType.list,
+        element_type=FieldType.integer,
+        description=[
+            "An array of key/value pairs representing unix timestamp"
+            + " and reading for outbound traffic."
+        ],
+    ),
+}
+
+linode_nodebalancer_stats_spec = dict(
+    state=SpecField(type=FieldType.string, required=False, doc_hide=True),
+    label=SpecField(type=FieldType.string, required=False, doc_hide=True),
+    connections=SpecField(
+        type=FieldType.list,
+        element_type=FieldType.integer,
+        description=[
+            "An array of key/value pairs representing unix timestamp"
+            + "a nd reading for connections to this NodeBalancer."
+        ],
+    ),
+    traffic=SpecField(
+        type=FieldType.dict,
+        suboptions=traffic_spec,
+        description=["Traffic statistics for this NodeBalancer."],
+    ),
+    title=SpecField(
+        type=FieldType.string,
+        description=[
+            "The title for the statistics generated in this response."
+        ],
+    ),
```

Comment:
Since these are all response fields, they can be excluded module spec 👍

Reply:
Does this mean that id is the only field that should be in the module spec?